agendas/2021-02-04.md

```diff
@@ -40,6 +40,7 @@ agenda, edit this file.*
 | Stephen Spalding ✏       | Netflix                  | Los Gatos, CA, US
 | Uri Goldshtein.          | The Guild                | Gan Yavne, IL
 | Morris Matsa             | IBM                      | Boston, US
+| James Baxley             |                          | South Carolina, US
```

Comment:
@jbaxleyiii We prefer that people provide a project or organization that they're associated with; if you don't have a suitable one right now perhaps put `GraphQL TSC`?